internal/data/data_manager.go

```diff
@@ -50,26 +50,26 @@ type dataManager struct {
 	blobStore
 
 	database          database.Plugin
-	publicstorage     publicstorage.Plugin
+	sharedstorage     sharedstorage.Plugin
 	exchange          dataexchange.Plugin
 	validatorCache    *ccache.Cache
 	validatorCacheTTL time.Duration
 }
 
-func NewDataManager(ctx context.Context, di database.Plugin, pi publicstorage.Plugin, dx dataexchange.Plugin) (Manager, error) {
+func NewDataManager(ctx context.Context, di database.Plugin, pi sharedstorage.Plugin, dx dataexchange.Plugin) (Manager, error) {
 	if di == nil || pi == nil || dx == nil {
 		return nil, i18n.NewError(ctx, i18n.MsgInitializationNilDepError)
 	}
 	dm := &dataManager{
 		database:          di,
-		publicstorage:     pi,
+		sharedstorage:     pi,
```

Comment:
Is this used? I see that blobstore (initialized below) uses its reference to sharedstorage, but I'm not sure this one actually gets used.